plugins/modules/nxos_pim_interface.py

```diff
@@ -60,8 +60,14 @@
     type: str
   hello_interval:
     description:
-    - Hello interval in milliseconds for this interface.
+    - Hello interval in milliseconds or seconds for this interface.
+    - Use the option I(hello_interval_ms) to specify if the given value is in
+      milliseconds or seconds. The default is seconds.
     type: int
+  hello_interval_ms:
+    description:
+    - Specifies that the hello_interval is in milliseconds.
```

Comment:
This was a bit confusing to me when I first looked at it.  Maybe add the following to the `description:` to make it explicitly clear?

`When set to True, this indicates that the user is providing the hello_interval in milliseconds so no conversion is required`

Reply:
+1